javascript/chubbyjs/app.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+const PathToRegexpRouteMatcher_1 = require("@chubbyjs/chubbyjs-framework-router-path-to-regexp/dist/PathToRegexpRouteMatcher");
```

Comment:
weird var names. is this the original source code ?

Reply:
No its not, its generated by `tsc` but the setup i tried didn't work so i put in the compiled version.

Reply:
you can put the original version and give me the command line you use to compile, I'll do it ❤️

Reply:
perfect, will do.
The command is `tsc` standard typescript compiler

Reply:
done